classes/woocommerce/class-admin-customers.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Automattic\Gravatar\GravatarEnhanced\Woocommerce;
+
+class AdminCustomers {
+	/**
+	 * @var string
+	 */
+	const FILTER_GRAVATAR_WC_ADMIN_CUSTOMERS_MODULE_ENABLED = 'gravatar_enhanced_wc_admin_customers_module_enabled';
+
+	/**
+	 * Check if the module is disabled by a filter.
+	 *
+	 * @return bool
+	 */
+	private function is_module_disabled() {
+		// Check if module is manually disabled by the filter.
```

Comment:
I realise this is also in the other WC module, but the wording and logic kind of feels like a double-negative. Flipping it around to `is_module_enabled` would allow you to just return the filter result directly.

Reply:
Agreed, I adapted this from `class-hovercards.php`, will refactor the WC ones.